test/test_tree.rb

```diff
@@ -1413,38 +1410,24 @@ def test_old_camelCase_method_names
                               isOnlyChild? nextSibling previousSibling nodeHeight nodeDepth
                               removeFromParent! removeAll! freezeTree! }
 
-      require 'structured_warnings'
-
-      DeprecatedMethodWarning.disable do
-        assert(@root.isRoot?)   # Test if the original method is really called
-      end
-
       meth_names_to_test.each do |meth_name|
-        assert_warn(DeprecatedMethodWarning) {@root.send(meth_name)}
-      end
-
-      # Special Case for printTree to avoid putting stuff on the STDOUT during the unit test.
-      begin
-        require 'stringio'
-        $stdout = StringIO.new
-        assert_warn(DeprecatedMethodWarning) { @root.send('printTree') }
```

Comment:
Previous version of the unit test checked that `printTree` method is available. New version doesn't check it anymore. It should arguably check it.